sklearn/ensemble/_gb.py

```diff
@@ -1494,9 +1495,6 @@ class GradientBoostingRegressor(RegressorMixin, BaseGradientBoosting):
     estimators_ : ndarray of DecisionTreeRegressor of shape (n_estimators, 1)
         The collection of fitted sub-estimators.
 
-    n_classes_ : int
```

Comment:
We need to keep this for the moment because we need to deprecate the attribute. So the idea is that if somebody was using `n_classes_` with this regressor we should return the expected value (`1`) but issue a warning. To do that we can use a `@property` in the class `GradientBoostingRegressor`:

```python
# FIXME: to be removed in 0.26
@deprecated("Attribute n_classes_ was deprecated in version 0.24 and "
            "will be removed in 0.26. Use 'classes_' instead")
@property
def n_classes_(self):
    return 1

Reply:
Then we will need a new test to ensure that the warning is raised when a user tries to access the deprecated attribute.
And finally, we need to add `.. deprecated::` thing in the docstring.

You can refer to https://scikit-learn.org/stable/developers/contributing.html#deprecation which give even more details and do not hesitate to ask questions.